worlds/sm64ex/__init__.py

```diff
@@ -76,7 +77,8 @@ def generate_early(self):
         self.topology_present = self.multiworld.AreaRandomizer[self.player].value
 
     def create_regions(self):
-        create_regions(self.multiworld, self.player)
+        self.subregion_map = {}
+        create_regions(self.multiworld, self.player, self.subregion_map)
```

Comment:
just pass the world and create/assign the dict there

Reply:
I'm going with N00byking's suggestion for implementation (as the world maintainer).